content/docs/0.8.1/deploy/upgrade/longhorn-manager.md

```diff
@@ -48,9 +48,12 @@ Therefore, the PVCs and PVs should be migrated to use the new CSI plugin for the
     ```
     kubectl get pv --output=jsonpath="{.items[?(@.spec.csi.driver==\"io.rancher.longhorn\")].spec.csi.volumeHandle}"
     ```
-
-2. Shut down the related workloads and detach the volumes. 
-3. Run this script for each volume:
+2. Remove finalizer `external-attacher/io-rancher-longhorn` for the related PV.
+    ```
+    kubectl edit pv <The corresponding PV of the volume found in step 1>
```

Comment:
Can we use the patch command to remove the finalizer instead?

Reply:
Since there is another PV finalizer `kubernetes.io/pv-protection` handled by Kubernetes and `kubectl patch` would remove all finalizers only,  I prefer to use `kubectl edit` here.